docs/api/modules/State-Management.md

```diff
@@ -0,0 +1,153 @@
+[👈 Return to Overview](../API.md)
+
+# State Management
+
+[Holocron Modules](../API.md#modules) rely on one Store provided by [Redux](https://redux.js.org/) used primarily to cache the results of loaded data and Modules. Every Module may add their own [Reducers](https://redux.js.org/basics/reducers/) to the shared Store (through [`holocronModule`](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#holocronmodule)) and may dispatch actions to transform the Store. Redux provides a simple way to store data on the Server and [preload the store](https://redux.js.org/recipes/server-rendering#inject-initial-component-html-and-state) used by the Browser. This is commonly referred to as [Server Side Rendering](https://redux.js.org/recipes/server-rendering). One App employs the [Ducks Specification](https://github.com/erikras/ducks-modular-redux) design pattern for the logical grouping of Reducers, Action Creators, and Selectors (e.g. the `error` ducks contains reducers, actions, and selectors for storing error data in the Redux Store).
+
+**Contents**
+* [Globals](#globals)
+* [Higher Order Components](#higher-order-components)
+* [Shared Ducks](#shared-ducks)
+
+## Globals
+
+### `global.BROWSER`
+
+**Runs On**
+* ✅ Server
+* ✅ Browser
+
+**Shape**
+
+```js
+global.BROWSER; // Boolean
+```
+
+The `global.BROWSER` is provided in the Server and Browser environments to determine if the code is currently being executed on the Server or the Browser.
+
+## Higher Order Components
+
+[Holocron Modules](../API.md#modules) use [Higher Order Components (HOC)](https://reactjs.org/docs/higher-order-components.html) to add behaviors regarding 1) when a Module loads, 2) connecting a Module with its [Reducer(s)](https://redux.js.org/basics/reducers/) to a Redux Store (similar to [Redux `connect`](https://react-redux.js.org/api/connect)) and 3) adding runtime validations for a Module.
```

Comment:
I'm not a fan of the inline numbering. Remove the numbers or make it an ol

Reply:
I really like inline numbering and I've used a few times elsewhere:
* https://github.com/americanexpress/one-app/pull/19/files/1f14dc8aabda592857072b3822bee43cb70de4b9#diff-16c8bdba86123da609a7dc1cdf2fccbbR321
* https://github.com/americanexpress/one-app/pull/19/files/1f14dc8aabda592857072b3822bee43cb70de4b9#diff-2919092b5b3a7dfbd26a8e11b7650597R29
* https://github.com/americanexpress/one-app/pull/19/files/1f14dc8aabda592857072b3822bee43cb70de4b9#diff-affc201ede6b1995b07d8f7a823a5dcdR5

It allows paragraph form to be more easily parsable to the eye and helpful in technical writing IMO. If you insist, Ill change it all.

Reply:
I find it less readable. Maybe a third opinion is needed

Reply:
I find it less readable as well. In my mind as I read it, the numbers end up blending in with everything else and if they are there to make a point, it doesn't get the point across to me.

Reply:
Democracy has spoken, I will remove the inline numbers.